projects/m1/032-taxi-fare/solutions/alessiacaporaso/taxi-fare.js

```diff
@@ -0,0 +1,14 @@
+const taxiKilometers = parseInt(prompt('Please, enter the distance traveled'))
+const baseFare = 4.00;
+let totalFare;
+// calc taki fares
+function calcTotalFare(kilometers){
+  if(kilometers <= 140){ // // if the distance is less than 140, fare is equaled base fare else add plus
+    totalFare = baseFare
+  } else {
+    totalFare = (((kilometers / 140) * 0.25)+ baseFare).toFixed(2)
+  }
+  return alert(`The cost is €${totalFare}`)
```

Comment:
Here I think you should return the value and print it outside the function.
`return totalFare`

`console.log(The cost is € ${calcTotalFare(taxiKilometers)}`